components/chip/src/chip.js

```diff
@@ -111,6 +115,8 @@ Chip.defaultProps = {
     marginBottom: 4,
     marginLeft: 4,
     marginRight: 4,
```

Comment:
Yes. Thanks! I think the current logic I have is wrong. I've updated it so that it should proceed in the following order
1) marginInlineStart (including 0)
2) marginLeft (including 0)
3) 4